fastlane/Fastfile

```
@@ -0,0 +1,86 @@
+  end
+
+  desc "Build and Deploy iOS Beta"
+  lane :build_deploy_beta do
+    xcodebuild(
+      project: IOS_PROJECT,
+      scheme: IOS_SCHEME,
+      configuration: "Release",
+      destination: "generic/platform=iOS Simulator",
+      xcargs: "KATANA_FLAVOR=beta",
+      build: true,
+      clean: true,
```

RuboCop (1.85.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)